### PR TITLE
Added Open Graph & Twitter Card tags, image

### PR DESCRIPTION
**Pull request by TrevorBramble**
Signed-off-by: Trevor Bramble <tbramble@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
By analyzing the blame information on this pull request, we identified @cwebberOps, @ryankeairns and @magwalk to be potential reviewers

**Comment by smith** [comment]
@TrevorBramble do you think we should add these to https://github.com/habitat-sh/habitat/blob/master/components/builder-web/index.html?

**Comment by TrevorBramble** [comment]
@smith Hm, I suppose so. Want me to include it here? (Feel free to push to this branch yourself if you want.)

**Comment by smith** [comment]
@TrevorBramble please do! Thanks!

File: components/builder-web/index.html [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
     <meta charset="utf-8">
     <meta http-equiv="x-ua-compatible" content="ie=edge">
     <title>Habitat - Automation that travels with the app.</title>
     <meta name="description" content="Habitat allows automation to travel with your app, so you can be confident that what you build, deploy and manage will behave consistently in any runtime.">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <meta property="og:title" content="Habitat - Automation that travels with the app.">
+    <meta property="og:type" content="website">
+    <meta property="og:image" content="/assets/images/habitat-social.jpg">
+    <meta property="og:url" content="https://www.habitat.sh/">
```
Comment (TrevorBramble) [inline components/builder-web/index.html]: @smith I don't know if it's important but all of the stats for shares under the app will be consolidated to this one URL. Don't know if there are Angularine ways to address that.

Comment (smith) [inline components/builder-web/index.html]: Yes let's leave it like this for now and perhaps we can figure out how to do fancier things later.

**Comment by smith** [comment]
gif-keyboard-4093673015185949059

@thesentinels r+

**Comment by thesentinels** [comment]
:pushpin: Commit d5df5a1 has been approved by `smith`

**Comment by thesentinels** [comment]
:hourglass: Testing commit d5df5a1 with merge e932b07...

**Comment by thesentinels** [comment]
:sunny: Test successful - [travis](https://travis-ci.com/habitat-sh/habitat/builds/26899529)

**Pull request merged**